open-reac/src/test/java/com/powsybl/openreac/parameters/input/ReportTestHelper.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
```

Comment:
I know the existing headers doesn't have it yet, but the SPDX clause should be present in our files.
I will later create a PR to had it to the existing files, but for now, we can still add it to new files.

```suggestion
/**
 * Copyright (c) 2025, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 */
```

Reply:
Got it for the next files! The change has been added in the following commit.